File: ios/CircularBuffer/CircularBufferPreviewViewManager.swift
import React

@objc(CircularBufferPreviewViewManager)
class CircularBufferPreviewViewManager: RCTViewManager {
    override func view() -> UIView! {
        return CircularBufferPreviewView()
    }

    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
}

File: android/app/src/main/java/com/felipenunes/pegueiapp/circularbuffer/CircularBufferPreviewViewManager.kt
package com.felipenunes.pegueiapp.circularbuffer

import com.facebook.react.uimanager.SimpleViewManager
import com.facebook.react.uimanager.ThemedReactContext
import com.facebook.react.uimanager.annotations.ReactProp

class CircularBufferPreviewViewManager : SimpleViewManager<CircularBufferPreviewView>() {

    override fun getName(): String = "CircularBufferPreviewView"

    override fun createViewInstance(reactContext: ThemedReactContext): CircularBufferPreviewView {
        return CircularBufferPreviewView(reactContext)
    }

    @ReactProp(name = "isActive")
    fun setIsActive(view: CircularBufferPreviewView, isActive: Boolean) {
        view.isActive = isActive
    }
}
